Library/Formula/nload.rb

```diff
@@ -11,6 +11,11 @@ class Nload < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
 
+  # F2 Option Crash and Wrong adapter read, see http://sourceforge.net/p/nload/bugs/8/
```

Comment:
I wrote to the author > 10 months ago and never heard back. Also you may find that that there were no new versions for a long time and source code in the CVS is older then one in the release.